xmtp_mls/src/groups/device_sync/consent_sync.rs

```diff
@@ -120,30 +128,25 @@ pub(crate) mod tests {
         // Have amal_a receive the message (and auto-process)
         let amal_a_sync_group = amal_a.get_sync_group(amal_a_conn).unwrap();
         assert_ok!(amal_a_sync_group.sync_with_conn(&amal_a_provider).await);
-
-        // Wait for up to 3 seconds for the reply on amal_b (usually is almost instant)
-        let start = Instant::now();
-        let mut reply = None;
-        while reply.is_none() {
-            reply = amal_b
+        xmtp_common::wait_for_some(|| async {
+            amal_b
                 .get_latest_sync_reply(&amal_b_provider, DeviceSyncKind::Consent)
                 .await
-                .unwrap();
-            if start.elapsed() > Duration::from_secs(3) {
-                panic!("Did not receive sync reply.");
-            }
-        }
+                .unwrap()
+        })
+        .await
+        .unwrap();
 
         // Wait up to 3 seconds for sync to process (typically is almost instant)
```

Comment:
might want to update this comment since `wait_for_eq` looks like it would wait for 60 sec